doc/man1/flux-alloc.rst

```diff
@@ -5,7 +5,6 @@
 flux-alloc(1)
 =============
 
-
```

Comment:
commit message:
"cross-reference tags for options are relative" :arrow_right: "...options that are relative"?

Reply:
Oh, actually it's the tags that are relative not the options.  I'll reword, it's an awkward sentence.

Reply:
Ah, now I see, sorry!